robotiq_force_torque_sensor/nodes/rq_sensor.cpp

```diff
@@ -50,13 +50,55 @@
 #include "robotiq_force_torque_sensor/ft_sensor.h"
 #include "robotiq_force_torque_sensor/sensor_accessor.h"
 
+typedef robotiq_force_torque_sensor::sensor_accessor::Request Request;
+
 static void decode_message_and_do(INT_8 const  * const buff, INT_8 * const ret);
+static void decode_message_and_do(robotiq_force_torque_sensor::sensor_accessor::Request& req,
+								  robotiq_force_torque_sensor::sensor_accessor::Response& res);
+
 static void wait_for_other_connection(void);
 static int max_retries_(100);
 
 ros::Publisher sensor_pub_acc;
 
 /**
+ * @brief decode_message_and_do Decode the message received and do the associated action
+ * @param req
+ * @param res
+ */
+static void decode_message_and_do(robotiq_force_torque_sensor::sensor_accessor::Request& req,
+								  robotiq_force_torque_sensor::sensor_accessor::Response& res)
+{
+	if (req.command_id == Request::COMMAND_SET_ZERO)
```

Comment:
Would this if statement and the one below be better combined into a case statement?  It took me a while to dissect the two of them.  I think a case statement would be clearer.